fbpic/main.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, Nz, zmax, Nr, rmax, Nm, dt,
                  n_order=-1, dens_func=None, filter_currents=True,
                  v_comoving=None, use_galilean=True,
                  initialize_ions=False, use_cuda=False,
-                 n_guard=None, n_damp=64, exchange_period=None,
+                 n_guard=None, n_damp=64, n_inject=None, exchange_period=None,
```

Comment:
I am wondering whether this parameter should be exposed in the API of the `Simulation` object. 
I think that it might confuse the users to have too many parameters (e.g. in the automatic documentation), and I think that there would very rarely be a case where we might want to change `n_inject`. 
@MKirchen What do you think about dropping this argument and always use the default number for `n_inject` (`n_guard/2`)

Reply:
Hey, that was my initial plan, but then I decided to explicitly add it to the Simulation object so that the user always has full control of the total number of cells if he wants (or at least can deduce from the parameters of the Simulation object what the total or local number of cells will be). Also if someone decides to use a custom exchange_period, he might want to use a custom number of injection cells as well. However, I do not have a strong opinion on this. I can also remove the parameter from the Simulation object.